algobot/helpers.py

```diff
@@ -40,6 +44,49 @@
 LONG_INTERVAL_MAP = {v: k for k, v in SHORT_INTERVAL_MAP.items()}
 
 
+class Paths:
```

Comment:
All path based information is encapsulated in this class

Reply:
We should probably store this class in another file. With that being said, we should really organize the layout of the project.. Everything is kinda all scattered all across right now

Reply:
I would advise perhaps doing that in another PR. Doing things incrementally and in small commits is easier. Especially to revert if things go south.